CONTRIBUTING.md

```diff
@@ -1,42 +1,36 @@
+* **join the bi-weekly calls**: Bi-weekly calls and conference call info is sent prior to the calls via the mailing list. If you join the mailing list, you should get a recurring invite at the beinning of each calendar year. Meeting minutes for the calls are in the [SPDX meetings repo](https://github.com/spdx/meetings/tree/master/legal); older meeting minutes can be found at http://wiki.spdx.org/
 
-Please do not raise general questions via issues in this Github repository, but use the appropriate workgroup mailing list.
+# Getting started 
+Below are some ways you can get started participating and contributing!
 
-## Request a new license or exception be added to the SPDX License List
-The SPDX Legal Team appreciates proposals for new free and open source licenses or exceptions to be added to the SPDX License List.  To request a license or exception be added to the SPDX License List, please:
+* Commenting on new license requests: Once you have familiarized yourself with the [License inclusion principles](DOCS/license-inclusion-principles.md), comment on new license requests as to whether you think the license should be included on the SPDX License List
 
-1.  Review the [license inclusion principles](DOCS/license-inclusion-principles.md).
+* Make suggestions to improvement for documentation: Newcomers have a great perspective as to the effectiveness of documentation! You can make suggestions via an issue, if you want to discuss the changes or if there is something specific that could be updated, then create a PR
 
-2.  Check the [SPDX License List](https://spdx.org/licenses/), keeping in mind the [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) and issues labeled [new license/exception request](https://github.com/spdx/license-list-XML/labels/new%20license%2Fexception%20request) to ensure this license or exception has not been previously requested.
+* Help prepare files for licenses approved to be added to the SPDX License List by following the [new license workflow](https://github.com/spdx/license-list-XML/blob/master/DOCS/new-license-workflow.md)
 
-3. Submit your request via the SPDX Online Tool [Submit New License](https://tools.spdx.org/app/submit_new_license/) using the guidance provided there. Also see the [explanation of the fields contained on the list](DOCS/license-fields.md).
-   - Note: You must have a Github account in order to use this tool/process.
-   - Note: Whether you use the [Submit New License](https://tools.spdx.org/app/submit_new_license/) tool or submit manually via a new issue, you will need to provide *all* of the information as per the form in the tool. Incomplete submissions will delay the process.
+* Review PRs 
+
+* Request a new license be added to the SPDX License List by following the instructions HERE ADDLINK
```

Comment:
```suggestion
* Request a new license be added to the SPDX License List by following the instructions in the [CONTRIBUTING](https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md) file.
```

Reply:
I left ADDLINK b/c the request new license info is moving off the Contributing doc and to it's own doc

Reply:
Makes sense. +1